zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/controller/CamundaOpenApiController.java

```diff
@@ -29,13 +34,16 @@ public ResponseEntity<String> loadCamundaApiDocs() {
       final ObjectMapper yamlReader = new ObjectMapper(new YAMLFactory());
       final Object obj = yamlReader.readValue(yamlContent, Object.class);
```

Comment:
❓ But isn't it quite expensive to always create this object mapper every time the static resource is requested? I am wondering if we can maybe do this ideally only once, even. It won't change, it's static.

Reply:
OK, looks like @danielkelemen just posted the same idea 😆

Reply:
I moved to the @PostConstruct the loading of the static file :) Great suggestion
as well moved the yaml mapepr to a Bean. I just pushed the updates

Thanks @danielkelemen  and @tmetzke  for the suggestions here
Great inputs <3